__init__.py

```diff
@@ -76,11 +81,32 @@ def _setup(self, force=False):
                     self.enable_fallback = \
                         self.settings.get('enable_fallback')
 
-    def _force_setup(self):
+                # Register tracker entities
+                self._register_tracker_entities()
+
+    def _force_setup(self) -> None:
         self.log.debug('Creating a new HomeAssistant-Client')
         self._setup(True)
 
-    def initialize(self):
+    def _register_tracker_entities(self) -> None:
+        """List tracker entities.
+
+        Add them to entity file and registry it so
+        Padatious react only to known entities.
+        Should fix conflict with Where is skill.
+        """
+        types = ['device_tracker']
+        entities = self.ha_client.list_entities(types)
+
+        if entities:
+            cache_dir = get_cache_directory(type(self).__name__)
+            self.tracker_file = pth_join(cache_dir, "tracker.entity")
```

Comment:
is this unique enough? what about prefixing it with `homeassistant`: "homeassistant.tracker.entity"?

Reply:
Cache dir is created per skill, so its ok.
```
cache_dir = get_cache_directory('CachingSkill')
```

Reply:
ah nice, didn't know that.